erts/doc/src/erl_cmd.xml

```diff
@@ -1035,6 +1035,62 @@ $ <input>erl \
           descriptor input events.
         </p>
       </item>
+      <tag><marker id="+JPcover"/><c>+JPcover true|false|function|function_counters|line|line_counters</c></tag>
+      <item>
+        <p>Since: OTP 27.0</p>
+        <p>Enables or disables support for coverage when running with
+        the JIT. Defaults to false.</p>
+        <taglist>
+          <tag><c>function</c></tag>
+          <item><p>All modules that are loaded will be instrumented to
+          keep track of which functions are executed. Information
+          about which functions that have been executed can be
+          retrieved by calling <seemfa
+          marker="kernel:code#get_coverage/2"><c>code:get_coverage(function, Module)</c></seemfa>.</p>
+	  </item>
+          <tag><c>function_counters</c></tag>
+          <item><p>All modules that are loaded will be instrumented to
+          count how many times each function is executed. Information
+          about how many times each function has been executed can be
+          retrieved by calling <seemfa
+          marker="kernel:code#get_coverage/2"><c>code:get_coverage(function,
+          Module)</c></seemfa>.</p>
+          </item>
+          <tag><c>line</c></tag>
```

Comment:
`+JPcover line` is rejected for me, whereas `+JPcover line_coverage` works. I don't know which of the documentation or the code is wrong here.

Reply:
The code was wrong. Fixed now.